src/App.js

```diff
@@ -8,9 +8,6 @@ function App() {
        <Router>
         <Routes>
           <Route path='/LandingPage' element={<LandingPage/>} />
```

Comment:
I suggest you edit your path namings to snake case and seperate each words with hyphen